app/src/main/java/org/sopt/dosopttemplate/api/ResponseGetFollowerDto.kt

```diff
@@ -0,0 +1,42 @@
+package org.sopt.dosopttemplate.api
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseGetFollwerDto(
```

Comment:
```suggestion
@Serializable
data class FollowerResponseDto(
    @SerialName("page")
    val page: Int,
    @SerialName("per_page")
    val perPage: Int,
    @SerialName("total")
    val total: Int,
    @SerialName("total_pages")
    val totalPages: Int,
    @SerialName("data")
    val data: List<User>,
    @SerialName("support")
    val support: Support,
) {
    @Serializable
    data class User(
        @SerialName("id")
        val id: Int,
        @SerialName("email")
        val email: String,
        @SerialName("first_name")
        val firstName: String,
        @SerialName("last_name")
        val lastName: String,
        @SerialName("avatar")
        val avatar: String
    )

    @Serializable
    data class Support(
        @SerialName("url")
        val url: String,
        @SerialName("text")
        val text: String
    )
}
```
저는 이런식으로 data class 계층을 주어서 종속되도록 표현해두었어요! 물론 이것은 취향차이입니다 ~~~